eth-contracts/contracts/ServiceProviderFactory.sol

```diff
@@ -611,6 +609,14 @@ contract ServiceProviderFactory is RegistryContract {
         return currentlyStakedForOwner;
     }
 
+    /// @notice Validate that the total service provider balance is between the min and max stakes for all their registered services
+    //          Validates that direct stake for sp is also above minimum
+    function validateAccountStakeBalance(address _sp)
```

Comment:
I basically made this wrapper function around validateAndGetAccountStakeBalance which swallows the return statement. i didn't want to duplicate the function without the return value because that's just silly. this seemed less silly, but still seems kind of silly. let me know what yall think

Reply:
looks like the crytic warning just moved to this function now?

Reply:
@SidSethi yeah exactly, that's the trade off here. we had six places where we didn't use a return value. so i basically exposed another external function that doesn't have a return value. so we have two getters for the same functionality, one with a return and one without, and we have one error instead of six.